docs/stdlib/sequence.rst

```diff
@@ -89,32 +89,34 @@ Sequences
 
 .. note::
```

Comment:
I had suggested some sweeping changes here that, upon reviewing the documentation more closely in context, didn't make a sense. What I would recommend is that, within the sections for `sequence_next` and `sequence_reset`, we add a paragraph specifically calling out this note and why it's relevant and linking to it.

Suggested callouts:

```rest
    See our notes on specifying the sequence to :eql:func:`sequence_next`.
```

and

```rest
    See our notes on specifying the sequence to :eql:func:`sequence_reset`.
```

In each paragraph, `our notes on specifying the sequence` should link to this point in the documentation.

Reply:
Sure, linking like that sounds OK.